src/Module.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @see       https://github.com/laminas/laminas-zendframework-bridge for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-zendframework-bridge/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-zendframework-bridge/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\ZendFrameworkBridge;
+
+use Laminas\ModuleManager\Listener\ConfigMergerInterface;
+use Laminas\ModuleManager\ModuleEvent;
+use Laminas\ModuleManager\ModuleManager;
+
+class Module
+{
+    /** @var string[] */
+    private $replacements;
```

Comment:
cannot see where it is used?

Reply:
Remnant of a previous implementation; I'll remove. Thanks!